packages/bits/demo/src/components/demo/paginator/paginator-visual-test/paginator-visual-test.component.html

```diff
@@ -42,4 +42,7 @@ <h3>Invisible Paginator</h3>
                        [adjacent]="1"
                        [hideIfEmpty]="true">
         </nui-paginator>
+
+        <h3>Paginator with Virtual scroll</h3>
```

Comment:
Title case. Minor nit:

```suggestion
        <h3>Paginator with Virtual Scroll</h3>
```